src/main/webapp/app/tasks/task-list/task-list.component.html

```diff
@@ -1,9 +1,48 @@
 <mat-list data-cy="task-list">
   <mat-list-item *ngFor="let task of tasks" class="mat-elevation-z1">
     <mat-icon mat-list-icon>assignment</mat-icon>
-    <h4 mat-line>{{task.name}}</h4>
+    <h4 mat-line>{{ task.name }}</h4>
     <button mat-icon-button aria-label="Delete task" color="primary">
-      <mat-icon aria-label="Delete task" (click)="delete(task)">delete</mat-icon>
+      <mat-icon aria-label="Delete task" (click)="delete(task)"
+        >delete</mat-icon
+      >
+    </button>
+    <label>
+      <input
+        aria-label="mark task as done"
+        type="checkbox"
+        [color]="'primary'"
+        [checked]="task.done"
```

Comment:
does not work because you made `done` string-typed, instead would need to write [checked]="task.done === 'done'"

Reply:
Yes careless mistake 🙈 But this would work, if I would use the boolean-attempt or?